application/views/template.php

```diff
@@ -45,18 +45,22 @@
 								echo $content;
 							}
 						} else {
-							$services_down = 'services LMD and Naemon are not running';
+							$services_down = '';
 
 							// Check LMD.
 							exec('service lmd status', $output, $return_var_lmd);
 							// Check Livestatus.
 							exec('service naemon status', $output, $return_var_le);
 
 							if($return_var_lmd !== 0 && $return_var_le === 0){
-								$services_down = 'service LMD is not running';
+								$services_down = 'The OP5 Monitor service LMD is not running, please contact your administrator.';
+							} elseif($return_var_lmd !== 0 && $return_var_le !== 0) {
+								$services_down = 'Issues encountered, check LMD and Naemon Logs for details.';
```

Comment:
Please check logic. 
This seems like LMD and naemon are down. 
And probably you should have a condition where naemon is down but LMD is up to complete all scenarios